### PR TITLE
Feature/team arch backend merge2

### PR DESCRIPTION
**Pull request by stanley-dejesus**
- Added Backend Features -> Google Vision OCR, Logo, & QR Detection
- Updated Images used for unit testing
- Added a separate route and button for accessing the backend test page. The button is located on the main menu page
- Updated Backend GUI to reflect Application UI appearance
- Below is the setup guide for obtaining your Cloud Vision credentials.

[How to setup Google Cloud Vision API and get credential key](https://github.com/umgc/summer2022/files/9076335/How.to.setup.Google.Cloud.Vision.API.and.get.credential.key.docx)

### CODE REVIEW

File: .flutter-plugins 2 [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# This is a generated file; do not edit or check into version control.
```
Comment (zsloan112) [inline .flutter-plugins 2]: Ensure this is added to the .gitignore file since it says it should not be checked in.

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
```
Comment (zsloan112) [inline .vscode/launch.json]: Recommend adding this to .gitignore as well.

File: android/local.properties [hunk 3]
```diff
@@ -1,5 +1,6 @@
-sdk.dir=C:\\Users\\zsloa\\AppData\\Local\\Android\\sdk
-flutter.sdk=C:\\flutter
+sdk.dir=C:\\SDK\\Android
```
Comment (zsloan112) [inline android/local.properties]: Believe this should be in .gitignore as well.

File: .flutter-plugins 2 (hunk 1 above)
Comment (zsloan112) [inline .flutter-plugins 2]: Add to gitignore and remove from source control

File: .idea/misc.xml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (zsloan112) [inline .idea/misc.xml]: Add to .idea folder to .gitignore and remove from source control

File: summer2022.iml [hunk 5]
```diff
@@ -1,18 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (zsloan112) [inline summer2022.iml:1]: If this file is empty, I would recommend deleting it and/or adding to .gitignore

**Review by zsloan112 — CHANGES_REQUESTED** [review]
Changes are very minor, mainly adding things to .gitignore that don't need to be in source control and deleting this from the source

**Review by zsloan112 — APPROVED** [review]
Looks good from my end

**Review by sriniva5 — APPROVED** [review]

**Pull request merged**